README.md

```diff
@@ -266,6 +268,39 @@ Example:
 kubergrunt eks sync-core-components --eks-cluster-arn EKS_CLUSTER_ARN
 ```
 
+#### cleanup-security-group
+This subcommand cleans up the leftover AWS-managed security groups that are associated with an EKS cluster you intend
+to destroy. It accepts
+- `--eks-cluster-arn`: the ARN of the EKS cluster
+- `--security-group-id`: the AWS-managed security group ID
+- `--vpc-id`: the VPC ID where the cluster is located
+
+It also looks for other security groups associated with the EKS cluster, such as the AWS-managed security group that
```

Comment:
> AWS-managed security group

I think this might be incorrect. We currently pass in the sg-id generated after creating the `vpc_config`, which might be the one that we manage.

Reply:
Yea the one the LB controller creates isn't really managed by AWS, but rather it is managed by Kubernetes Services. I might phrase this as:

```
... such as the security group created by the AWS Load Balancer Controller.
```